install.ps1

```diff
@@ -0,0 +1,186 @@
+# Get-ExecutionPolcy = "Restricted" by default
+
+# VARIABLES, CHANGE AS NEEDED
+$GITHUB_REPO="https://github.com/bmFtZQ/edge-frfox.git"
+$PROJECT_NAME=$($GITHUB_REPO.Split("/")[-1] -replace ".git", "")
+$FIREFOX_DIR="$env:APPDATA\Mozilla\Firefox";
+$PROFILE_ROOTDIR="$($FIREFOX_DIR)\Profiles\$((Select-String -Path "$($FIREFOX_DIR)\profiles.ini" -Pattern "Path=.*\.(dev-edition-default|default-.*)" | Select-Object -Last 1).Line.Substring(14))";
+$OPTIONALS=@{
+  "browser.theme.dark-private-windows" = "false"
```

Comment:
While we're here, add the new optional preference for Windows: `browser.theme.windows.accent-color-in-tabs.enabled` = `true`.